cmd/faucet/faucet.go

```diff
@@ -492,6 +492,24 @@ func (f *faucet) apiHandler(conn *websocket.Conn) {
 	}
 }
 
+func (f *faucet) setNonceAndPrice() {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+	nonce, err := f.client.NonceAt(ctx, f.account.Address, nil)
```

Comment:
This code runs on every block. What if for some reason we receive an error? probably the `price` and `nonce` will be 0 because it's an error. Do we want to update them to values coming from an error?

Reply:
I'd say that we can leave like they did for now.